### PR TITLE
Wdb 62/configure password encryption

### PR DESCRIPTION
**Pull request by kibartas**
On to implementing the Login endpoint

### CODE REVIEW

File: backend/backend/Models/User.cs [hunk 1]
```diff
@@ -23,6 +23,9 @@ public User()
 
         [Required]
         public string Password { get; set; }
+        
+        [Required]
```
Comment (AArlauskas) [inline backend/backend/Models/User.cs]: Please, kind sir, could you add salting to the constructor?

**Review by AArlauskas — CHANGES_REQUESTED** [review]

File: backend/backend/Models/User.cs (hunk 1 above)
Comment (kibartas) [reply backend/backend/Models/User.cs]: I'd be glad to

**Review by AArlauskas — APPROVED** [review]
Nice job Salt bae

**Review by salmonela7 — APPROVED** [review]
Looks good. 👍

**Pull request merged**